packages/hardhat-core/sample-projects/javascript-esm/ignition/modules/LockModule.js

```diff
@@ -0,0 +1,19 @@
+// ./ignition/LockModule.js
+import { buildModule } from "@nomicfoundation/hardhat-toolbox";
+
+const currentTimestampInSeconds = Math.round(new Date(2023, 0, 1) / 1000);
+const TEN_YEAR_IN_SECS = 10 * 365 * 24 * 60 * 60;
+const TEN_YEARS_IN_FUTURE = currentTimestampInSeconds + TEN_YEAR_IN_SECS;
+
+const ONE_GWEI = 1_000_000_000n;
+
+export default buildModule("LockModule", (m) => {
+  const unlockTime = m.getParameter("unlockTime", TEN_YEARS_IN_FUTURE);
```

Comment:
I am going to remove the default unlock time.
Ignition modules shouldn't have default parameters that change, as that is guaranteed to fail reconciliation between runs if no parameter is specified.